app/views/lobby/bits.scala

```diff
@@ -95,25 +95,25 @@ object bits {
     )
 
   def playbanInfo(ban: lila.playban.TempBan)(implicit ctx: Context) = nopeInfo(
-    h1("Sorry :("),
-    p("We had to time you out for a ", (ban.remainingSeconds < 3600) ?? "little ", "while."),
-    p("The timeout expires ", strong(secondsFromNow(ban.remainingSeconds)), "."),
-    h2("Why?"),
+    h1(trans.sorry()),
+    p(trans.weHadToTimeYouOutForAWhile()),
+    p(trans.timeoutExpires(), " ", strong(secondsFromNow(ban.remainingSeconds)), "."),
```

Comment:
this is called i18n legos: you can't concatenate stuff like this, it won't work in many languages. Use variables instead.